### PR TITLE
Update dlp-policy-reference.md

### PR DESCRIPTION
**Pull request by Pulkit-Sachan**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 1]
```diff
@@ -279,9 +279,9 @@ The available context options change depending on which location you choose. If
 - Content contains
 - Content is shared from Microsoft 365
 - Document created by
-- Document created by member of
+- Document created by member of - Currently deprecated for customers not already using this predicate.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md]:
```suggestion
- Document created by member of (currently deprecated for customers not already using this predicate) 
```

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 2]
```diff
@@ -279,9 +279,9 @@ The available context options change depending on which location you choose. If
 - Content contains
 - Content is shared from Microsoft 365
 - Document created by
-- Document created by member of
+- Document created by member of - Currently deprecated for customers not already using this predicate. 
 - Document name contains words or phrases
-- Document name matches patterns
+- Document name matches patterns - Currently deprecated for customers not already using this predicate.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md]:
```suggestion
- Document name matches patterns (currently deprecated for customers not already using this predicate) 
```

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 3]
```diff
@@ -291,9 +291,9 @@ The available context options change depending on which location you choose. If
 - Content contains
 - Content is shared from Microsoft 365
 - Document created by
-- Document created by member of
+- Document created by member of - Currently deprecated for customers not already using this predicate.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md]:
```suggestion
- Document created by member of (currently deprecated for customers not already using this predicate) 
```

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 4]
```diff
@@ -291,9 +291,9 @@ The available context options change depending on which location you choose. If
 - Content contains
 - Content is shared from Microsoft 365
 - Document created by
-- Document created by member of
+- Document created by member of - Currently deprecated for customers not already using this predicate. 
 - Document name contains words or phrases
-- Document name matches patterns
+- Document name matches patterns - Currently deprecated for customers not already using this predicate.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md]:
```suggestion
- Document name matches patterns (currently deprecated for customers not already using this predicate) 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Pulkit-Sachan, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Pull request merged**